src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -364,8 +364,9 @@ public static class Importer
          public static readonly string TheUnitInformationMustBeEnteredOrConfirmed = "The unit information must be entered or confirmed.";
          public static readonly string TheMetaDataInformationMustBeEnteredOrConfirmed = "The meta data must be entered or confirmed.";
          public static readonly string ResetMapping = "Reset Mapping";
-         public static readonly string ResetMappingBasedOnCurrentSheet = "Reset Mapping based on current sheet";
-         public static readonly string ClearMapping = "Clear Mapping";
+         public static readonly string ResetMappingToolTip = "Automatically recalculates the format based on the sheet currently selected and sets the Mapping Settings accordingly.";
```

Comment:
The case is weird
Mapping Settings => mapping settings
GRid=> grid

Reply:
done